pkg/model/api/v1/dashboard_test.go

```diff
@@ -285,7 +286,8 @@ func TestMarshalDashboard(t *testing.T) {
                 "up"
               ]
             }
-          }
+          },
+          "queries": null
```

Comment:
if you pull the new changes where i add `omitempty`, it should undo this change, right?

Reply:
yeah totally, this should disappear !

Reply:
fixed :)